politeiad/cache/cockroachdb/decred.go

```diff
@@ -157,50 +178,58 @@ func (d *decred) voteOptionResults(options []VoteOption) ([]VoteOptionResult, er
 func (d *decred) voteResultsMissing(bestBlock uint64) ([]string, []string, error) {
 	// Find standard vote proposals that have finished voting but
 	// have not yet been added to the VoteResults table.
-	q := `SELECT start_votes.token
+	standard := make([]string, 0, 1024)
+	runoff := make([]string, 0, 1024)
+	lbb := d.bestBlockGet()
+	// Only update the table if this procedure has not been run for
+	// the received best block.
+	if lbb != bestBlock {
```

Comment:
What's cleaner, nesting this entire function in an if statement or just returning?  Go with the cleaner, easier to read solution.
```
	if d.bestBlockGet() == bestBlock {
		// Vote results tables have already been built for this block.
		// There is no need to run these queries.
		return []string{}, []string{}, nil
	}
```

Reply:
Also, what's the reason for initializing the lbb variable instead of writing in inline?  If there's a reason to initialize a new variable, like its being used multiple times or the line goes over the 80 col limit, then it makes sense, but in this case its only being used once and it can be written inline without breaking any style rules.

Reply:
Indeed much cleaner, thanks for pointing these out .